aten/src/ATen/native/cuda/ReduceMinMaxKernel.cu

```diff
@@ -74,6 +74,10 @@ void argmax_kernel_cuda(TensorIterator& iter) {
     // Instead of implementing is_nan and warp_shfl_down
     // we can convert halves to float and do all the operations in float
     argmax_kernel_cuda_impl<at::Half, float>(iter);
+  } else if (iter.dtype(1) == kBFloat16) {
+    // Instead of implementing is_nan and warp_shfl_down
+    // we can convert halves to float and do all the operations in float
```

Comment:
This would be "convert bfloat16 to float"

Reply:
Thanks for the catch! I'll edit the comment when I'll push next time.